jetstream/jetstream_options.go

```diff
@@ -104,6 +104,9 @@ func WithGetMsgSubject(subject string) GetMsgOpt {
 // PullMaxMessages limits the number of messages to be buffered in the client.
 // If not provided, a default of 500 messages will be used.
 // This option is exclusive with PullMaxBytes.
+//
+// PullMaxMessages implements PullConsumeOpt and PullMessagesOpt and thus can
+// be used to configure both Consumer.Consume and Consumer.Messages.
```

Comment:
```suggestion
// PullMaxMessages implements both PullConsumeOpt and PullMessagesOpt, allowing it to configure Consumer.Consume and Consumer.Messages.
```

How about such phrasing?

Reply:
Done